docs/en/manuals/extensions_details.md

```diff
@@ -0,0 +1,79 @@
+
+# The Defold build setup
+
+Here we list some relevant build information, in order to make the integrations with your extensions as easy as possible.
+
+Here are some things to consider when you create an extension for the Defold engine.
+
+# Static linkage
+
+The custom engine is built using static linkage.
+The main reason is that on iOS, multiple executable binaries in an .ipa aren't allowed in the app store.
+
+# No Exceptions:
+
+We don't make use of any exceptions in the engine, as it decreases executable size and improves the runtime performance. This may be beneficial for you to if you wish to get a minimal extension.
```

Comment:
"Syftningsfel" maybe? :) At first read through it sounded like adding exceptions results in decreased exe size and improved runtime performance. Perhaps it can be reworded?